modules/nf-core/find/unpigz/main.nf

```diff
@@ -0,0 +1,74 @@
+process FIND_UNPIGZ {
+    tag "${meta.id}"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container
+        ? 'https://depot.galaxyproject.org/singularity/pigz:2.8'
+        : 'biocontainers/pigz:2.8'}"
+
+    input:
+    tuple val(meta), path(files_in)
+
+    output:
+    tuple val(meta), path("${prefix}.*"), emit: file_out
+    path "versions.yml", emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ""
+    prefix = task.ext.prefix ?: "${meta.id}"
+
+    file_extensions = files_in.collect { in_file -> in_file.name - in_file.getBaseName(in_file.name.endsWith('.gz') ? 2 : 1) }.toSet()
+
+    file_names = files_in.collect { it.toString() }
+
+    pattern_string = generatePatternString(file_extensions.toList())
+
+    if (!file_extensions.every { it.endsWith(".gz") }) {
+        error("All files provided to this module must be gzipped (and have the .gz extension).")
+    }
+
+    if (file_names.any { it.startsWith("${prefix}") }) {
+        error("No input files can start with the same name as the output prefix in the module FIND_UNPIGZ (currently '${prefix}'). Please choose a different one.")
+    }
+
+    """
+    find . -maxdepth 1 \\( -not -name '.*'  ${pattern_string} \\) |\\
+        sed 's:^./::g' | sed 's/.gz\$//g' | xargs -I{} sh -c "unpigz -cd --processes ${task.cpus} ${args} {}.gz > ${prefix}.{}"
```

Comment:
```suggestion
    find . ${args} -maxdepth 1 \\( -not -name '.*'  ${pattern_string} \\) |\\
        sed 's:^./::g' | sed ${args2} 's/.gz\$//g' | xargs -I{} sh -c "unpigz -cd --processes ${task.cpus} ${args3} {}.gz > ${prefix}.{}"
```
Missing args for each part of the pipe [^1]

[^1]:  https://nf-co.re/docs/guidelines/components/modules#use-of-multi-command-piping